pkg/server/server.go

```diff
@@ -366,6 +373,20 @@ func WithCacheLimit(limit uint32) OpenFGAServiceV1Option {
 	}
 }
 
+// WithCacheControllerEnabled enables cache invalidation of different cache entities.
+func WithCacheControllerEnabled(enabled bool) OpenFGAServiceV1Option {
```

Comment:
We could use some simple tests to validate that this wires up things correctly down the road:

```
if s.cache != nil && s.cacheControllerEnabled {
```

Reply:
Tests are actually included already.